src/common/interfaces/IGateway.sol

```diff
@@ -41,6 +41,11 @@ interface IGateway is IMessageHandler, IMessageSender, IGatewayHandler {
         bytes pendingBatch;
     }
 
+    struct Funds {
+        uint256 value;
```

Comment:
Should we store `value` as `uint64` (and just discard remaining decimals), such that this struct is packed into 1 storage slot?